src/components/CustomCheckbox/custom_checkbox.module.scss

```diff
@@ -8,17 +8,16 @@
   padding-top: rem(0.8);
   z-index: 0;
   margin-bottom: auto;
+  gap: 2rem;
```

Comment:
@agrim-deriv let's keep it consistent with other values, make it `rem(2)` as above padding-top property